lib/service/connect.go

```diff
@@ -949,8 +948,8 @@ func (process *TeleportProcess) newClientDirect(authServers []utils.NetAddr, tls
 			return nil, trace.Wrap(err)
 		}
 		dialOpts = append(dialOpts, []grpc.DialOption{
-			grpc.WithChainUnaryInterceptor(metadata.UnaryClientInterceptor, om.UnaryClientInterceptor(grpcMetrics)),
-			grpc.WithChainStreamInterceptor(metadata.StreamClientInterceptor, om.StreamClientInterceptor(grpcMetrics)),
+			grpc.WithUnaryInterceptor(om.UnaryClientInterceptor(grpcMetrics)),
+			grpc.WithStreamInterceptor(om.StreamClientInterceptor(grpcMetrics)),
```

Comment:
Note to reviewers: The metadata interceptors are intentionally being removed here. They are already included in the default set of interceptors used by the api client.